### ISSUE TITLE
:open autocomplete does not show `javascript:` links

### ISSUE THREAD

**Issue by ratijas** [vimperator/vimperator-labs#336]
discovered that i can't launch javascript bookmarks with vimperator.  maybe other protocols are affected as well.

**Comment by timss** [comment]
Test bookmarklet: `javascript:(function(){ window.open('http://vimperator.org'); })();`

Funny thing is that it's actually visible using the `:bmarks`, but once selected it's not considered a proper URL and thus not available to be opened using for example `;o`. Haven't checked myself, but the same could be true for `:open` etc.

**Comment by ratijas** [comment]
yes, i can see it in `:bmarks`,
no, i cannot open it with `:open` family.

by the way, `Y` and `;o` never works for me in such overlaying messages as `:bmarks` produces.

**Comment by timss** [comment]
I don't have much experience with bookmarks in Vimperator, but the map seems to be `;y`, not `Y` as described in the help message. Other extended hint modes such as `;t` work as well. Might have to look into updating the help message.

**Comment by ratijas** [comment]
no, `;y` does not work as well.  it just self-closes, like if there's nothing to `hint`.

**Comment by timss** [comment]
Works for me using FF 44.0.1 and Vimperator 3.12.0. What are you using?

**Comment by ratijas** [comment]
FF 44.0.2, one of latest Vimperator, Mac OS X 10.8.2.

never worked since i became a user in august 2015.

**Comment by timss** [comment]
Sorry. I'm still not able to reproduce. Can you please try with a fresh profile, and ensure that you got the latest [release](https://github.com/vimperator/vimperator-labs/releases/tag/vimperator-3.12.0)?

    # create a new profile
    $ firefox -no-remote -P

    # start with no config
    $ firefox -no-remote -P <new profile> -vimperator "+u NONE"

Check using one of the standard bookmarks, https://www.mozilla.org/en-US/firefox/help/ or something.
